app/test_app/unit/utils/ussd/test_token_processor.py

```diff
@@ -32,8 +32,8 @@ def create_transfer_account_for_user(user: User, token: Token, balance: float, i
 
 
 @pytest.mark.parametrize("user_type,limit,preferred_language,sample_text", [
-    ("standard", None, "en", "Your Sarafu-Network balances are as follows: SM1 200.00\nSM2 350.00\nCall 0757628885 for more info"),
-    ("standard", None, "sw", "Akaunti yako ya Sarafu-Network ina salio yafuatayo: SM1 200.00\nSM2 350.00\nPiga 0757628885 kwa usaidizi zaidi"),
+    ("standard", None, "en", "Your balances are as follows: SM1 200.00\nSM2 350.00\nCall 0757628885 for more info"),
```

Comment:
We should remove this phone number from all references for now. 

In the future, we could look into adding a hotline contact number to the organization table for all outbound SMS's